Server/ChatService/services/call_service.py

```
@@ -0,0 +1,199 @@
+            if websocket_manager:
+                try:
+                    await websocket_manager.send_call_event_to_user(
+                        call_data["caller_id"],
+                        "call_ended",
+                        {
+                            "call_id": call_id,
+                            "reason": "didn't_answer"
+                        }
+                    )
+                    print(f"[CallService] Notified caller {call_data['caller_id']} that call wasn't answered")
+                except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)